src/test/java/teammates/test/pageobjects/InstructorFeedbackEditPage.java

```diff
@@ -1311,6 +1319,13 @@ public void setMsqMinSelectableChoices(int qnNumber, int value) {
         executeScript(String.format("$('#%s').val(%d);$('#%s').change();", id, value, id));
     }
 
+    public void fillMsqMinSelectableChoices(int qnNumber, String value) {
+        assertTrue(isMsqMinSelectableChoicesEnabled(qnNumber));
+
+        WebElement inputBox = getMsqMinSelectableChoicesBox(qnNumber);
+        fillTextBox(inputBox, value);
+    }
+
```

Comment:
@LiHaoTan, do you think we should replace `setMsqMaxSelectableChoices` with this method?

Reply:
@Crphang @LiHaoTan On a second thought, I think we should not replace `setMsqMaxSelectableChoices` with this method, the reason being that `setMsqMaxSelectableChoices` is more appropriate to set value in this number field, as it takes `int` instead of String, the `fill*` is only convenient when checking for invalid inputs like empty string or a string in a number field. But other than that, it is not intuitive to pass a string in a number field, and it will be a hassle for the developer to keep this use case in mind whenever using this method. 

How about instead of introducing a new method to fill the number field with an empty field, I just do this:

`fillTextBox(getMsqMinSelectableChoicesBox(qnNumber), "");`

Reply:
The problem with `setMsqMaxSelectableChoices` is that it deliberately omits the fact that a string is actually allowed, entirely skipping the situation a user can cause an error so a string is actually more appropriate.

> fillTextBox(getMsqMinSelectableChoicesBox(qnNumber), "");
I do agree that in general we should have abstractions that are more general than specific so unless in cases where such a method need to be reused frequently. We can avoid a lot of duplication if they are less specific.

Reply:
Can always overload the method so it accepts both `int` and `String` for convenience?

Reply:
>Can always overload the method so it accepts both `int` and `String` for convenience?

@whipermr5 Yes, this can do. But I was wondering will it be used that often? In any case, should I make the changes. Will do after your confirmation.

Reply:
I'll leave it to @LiHaoTan to decide

Reply:
I suppose overloading can be awkward if it's just for convenience since inputs in HTML always only accept strings.

Anyhow, since the purpose of this PR is just to add front-end validation, I think it's fine to keep `setMsqMaxSelectableChoices` as well.

Though of course, I think there are many reasons such a method can be useful:
* Used commonly
* Serious disadvantages to using `String`, for instance, e.g. a field that genuinely accepts only numbers but due to implementation limitations it was implemented using strings
* Etc

I guess the intuition is that `input type="number"` should accept numbers only although that isn't really the case, and I would say it is also related to how form data is also a string.

All the related abstractions lead me to believe using `String` as the data type will be simpler although as mentioned I think there can be good reasons not to.

And if we relook at this issue, it's because we wrongly abstracted a String(which can be empty, or even non-digits) to an integer (that can never be empty) that we get such an error.

Reply:
@LiHaoTan Just to avoid misinterpretation just wanted to be sure that I understood your message correctly. You are suggesting not to overload `setMsqMaxSelectableChoices` and just change 
`setMsqMaxSelectableChoices(WebElement inputBox, int value)` to `setMsqMaxSelectableChoices(WebElement inputBox, String value)`. am I right?

Reply:
@LiHaoTan Waiting for your confirmation on this.

Reply:
@sukanta-27 I'm saying we can keep `setMsqMaxSelectableChoices(WebElement inputBox, int value)` since it's not particularly important here.